apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx

```diff
@@ -192,33 +195,36 @@ class UnitOverviewTopRow extends React.Component {
                 unitId={scriptId}
               />
             )}
-          {pdfDropdownOptions.length > 0 && viewAs === ViewType.Instructor && (
-            <div style={{marginRight: 5}}>
-              <DropdownButton
-                customText={
-                  <div>
-                    <FontAwesome icon="print" style={styles.icon} />
-                    <span style={styles.customText}>
-                      {i18n.printingOptions()}
-                    </span>
-                  </div>
-                }
-                color={Button.ButtonColor.blue}
-              >
-                {pdfDropdownOptions.map(option => (
-                  <a
-                    key={option.key}
-                    href={option.url}
-                    onClick={e =>
-                      this.recordAndNavigateToPdf(e, option.key, option.url)
-                    }
-                  >
-                    {option.name}
-                  </a>
-                ))}
-              </DropdownButton>
-            </div>
-          )}
+          {pdfDropdownOptions.length > 0 &&
```

Comment:
Nit (readability / style): I would recommend pulling this boolean logic (from line 198 to 201) into a variable outside of the return statement, up by `let completedProfessionalLearningCourse = . . . `. As a code reader, that lets me front-load all the logic in this function and simplify what is in the return statement.

Reply:
That makes sense - I wasn't sure if you were suggesting including the`ViewType.Instructor` logic too in the variable.  Given that all the other code had that separated out, I didn't include it in the variable.